### PR TITLE
 add Slow Falling Effect in PMMP 4.0

### PR DESCRIPTION
**Pull request by SenseiTarzan**
add Slow Falling and remove fall damage entity if has Slow Falling

### CODE REVIEW

File: src/entity/effect/VanillaEffects.php [hunk 1]
```diff
@@ -58,6 +58,7 @@
  * @method static Effect WATER_BREATHING()
  * @method static Effect WEAKNESS()
  * @method static WitherEffect WITHER()
+ * @method static SlowFallingEffect Slow_Falling()
```
Comment (buchwasa) [inline src/entity/effect/VanillaEffects.php:61]: You shouldn't manually write this, it's autogenerated

Comment (SenseiTarzan) [reply src/entity/effect/VanillaEffects.php:61]: How does it

Comment (SenseiTarzan) [reply src/entity/effect/VanillaEffects.php:61]: How does it work?

Comment (dktapps) [reply src/entity/effect/VanillaEffects.php:61]: https://github.com/pmmp/PocketMine-MP/blob/master/build/generate-registry-annotations.php

Comment (SenseiTarzan) [reply src/entity/effect/VanillaEffects.php]: it's good I just did with generate-registry-annotations for the annatotion of the sloaw falling in the vanillaEffect and the change of the names after the generation of the method

**Comment by dktapps** [comment]
This also needs to be registered in PotionType enum: https://github.com/pmmp/PocketMine-MP/blob/07d97bbdeb891a019f11272b0a10a23f135180de/src/item/PotionType.php#L198
https://github.com/pmmp/PocketMine-MP/blob/07d97bbdeb891a019f11272b0a10a23f135180de/src/item/PotionType.php#L201

**Comment by SenseiTarzan** [comment]
ok I show that there is no problem

**Comment by dktapps** [comment]
Closing due to lack of activity on review comments.

**Pull request closed without merging**